articles/resources/relnotes.md

```diff
@@ -17,6 +17,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: August 26th, 2020*
+
+This release contains the following:
+
+- Fixed bug in M.Q.Arithmetic.RippleCarryAdderCDKM.
+- Fixed intermittent QDK build failures due to access violation: "Test host process crashed".
```

Comment:
I think this one should be phrased differently to reflect the potential benefit to users, rather than the impact on our internal build pipeline. @swernli do you have a suggestion here?

Reply:
If this was part of the bug bash, we agreed we will just mention bug fixes in general in release notes